pom.xml

```diff
@@ -79,15 +79,16 @@
         <apache.kafka.version>3.5.1</apache.kafka.version>
         <apache.ranger.version>2.0.0</apache.ranger.version>
         <apache.ranger.gson.version>2.2.4</apache.ranger.gson.version>
-        <scala.library.version>2.13.11</scala.library.version>
+        <scala.library.version>2.13.9</scala.library.version>
```

Comment:
this seems wrong, why are we downgrading the versions compared to what is in 28?